src/Imdb/Title.php

```diff
@@ -1262,13 +1262,15 @@ public function sound()
     public function mpaa()
     {
         if (empty($this->mpaas)) {
-            $this->getPage("ParentalGuide");
-            if (preg_match_all("|/search/title\?certificates=.*?>\s*(.*?):(.*?)<|", $this->page["ParentalGuide"],
-                $matches)) {
-                $cc = count($matches[0]);
-                for ($i = 0; $i < $cc; ++$i) {
-                    $this->mpaas[$matches[1][$i]] = $matches[2][$i];
-                }
+            $xpath = $this->getXpathPage("ParentalGuide");
+            if (empty($xpath)) {
+                return array();
+            } // no such page
+            $cells = $xpath->query("//section[@id=\"certificates\"]//li[@class=\"ipl-inline-list__item\"]");
+            foreach ($cells as $cell) {
+                $value = $cell->getElementsByTagName('a')->item(0)->nodeValue;
```

Comment:
Would like a check for if item(0) are !== null before using nodeValue. At the moment it can crash.

Reply:
jep agree, i thought about it, wasn't sure how to check